lib/crypto.js

```diff
@@ -666,25 +664,25 @@ function Certificate() {
 }
 
 
-Certificate.prototype.verifySpkac = function verifySpkac(object) {
-  return binding.certVerifySpkac(object);
+Certificate.prototype.verifySpkac = function verifySpkac(spkac) {
+  return binding.certVerifySpkac(spkac);
 };
 
 
 Certificate.prototype.exportPublicKey =
-  function exportPublicKey(object, encoding) {
-    return binding.certExportPublicKey(toBuf(object, encoding));
+  function exportPublicKey(spkac, encoding) {
+    return binding.certExportPublicKey(toBuf(spkac, encoding));
   };
 
 
 Certificate.prototype.exportChallenge =
-  function exportChallenge(object, encoding) {
-    return binding.certExportChallenge(toBuf(object, encoding));
+  function exportChallenge(spkac, encoding) {
+    return binding.certExportChallenge(toBuf(spkac, encoding));
   };
 
 
-exports.setEngine = function setEngine(id, flags) {
-  if (typeof id !== 'string')
+exports.setEngine = function setEngine(engine, flags) {
+  if (typeof engine !== 'string')
     throw new TypeError('"id" argument should be a string');
```

Comment:
This TypeError gets really weird now.

Also, if anything, `engineID` is probably better.